### PR TITLE
Changes in README.md.

### PR DESCRIPTION
**Pull request by rautarchana9**
Edits the README.md for installing HasCore.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,10 +7,49 @@ HasCore: HasGeek API
 Hascore provides helper API methods including the networkbar that is shown
 on all sites, and API endpoints for geolocation and language analysis.
 
-Hascore requires NLTK. You will need to install NLTK's corpus data.
-Be warned, this is about 1.8 GB:
+##How to install HasCore on your machine.
```
Comment (jace) [inline README.md]: Hascore has a lowercase `c`. Please amend everywhere the name is used.

**Comment by sindhus** [comment]
Also correct grammar by proof reading before sending PR. Changes *to* readme or "Update readme" followed up empty line and then a brief explanation of what's changed in the file. For example: [Hasjob readme update](https://github.com/hasgeek/hasjob/commit/b4d3576987de25985f84e01d85b8a6deea99fc14)

**Comment by rautarchana9** [comment]
Sure, thank you for the feedback. I make the necessary changes.

Archana

**Comment by rautarchana9** [comment]
1. ### Sets default lang to "en".

    Geonames database has duplicates based on lang, besides current DB has "iata" set as a lang for iata         city codes. 
    e.g. GOA is iata code for Genoa, and shows up in search if lang is not set.

2. ### Changes parse_locations function to calculate distance between two tokens and sorts by shortest distance.
    Takes db search results for each token provided by user, calculates the geographical distance             between the two tokens (places) by a function called `distance`, which takes latitude and longitude as parameters and returns distance in kms. The relatedness of tokens is inversely proportional to the geographical distance between them.

**Comment by sindhus** [comment]
This PR is about changes to Readme.md, submitting a comment about changes in another branch about another piece of work is just confusing. Make sure you don't have unrelated work in a branch before you push and send a PR.

**Comment by rautarchana9** [comment]
So should I create a new pull request for these changes ?

Archana

**Comment by rautarchana9** [comment]
I have deleted the branch map_score, current changes are on this branch,
master.

**Comment by sindhus** [comment]
Yes at this point, this PR is messed up with unrelated work. I suggest you check on your computer:

* Do I have a branch that has __only__ changes to Readme.md? If yes, push this branch to your repo (hasjob fork) and send PR
* Do I have a branch that has __only__ changes to geonames related work? If yes, push this branch to your repo (hasjob fork) and send PR

If not then, ask someone to show you how to untangle your mixed up work with `git format-patch HEAD~1`.

Always create new branches for submitting PR. Do not commit work to your master. Your master branch should always be a copy of hasgeek/hasjob master.

**Comment by rautarchana9** [comment]
I have made different branches for diffrent changes now.
Thanks for the suggestions.

Archana

**Pull request closed without merging**